jekyll/_cci2/gh-bb-integration.md

```diff
@@ -328,3 +328,33 @@ access to multiple repositories) and user keys have no notion of _scope_
 separate from the user associated with them.
 
 To achieve fine-grained access to more than one repo, consider creating what GitHub calls a machine user. Give this user exactly the permissions your build requires, and then associate its user key with your project on CircleCI.
+
+## Authenticity of a host can't be established
```

Comment:
Maybe we can change this heading to be positive? "Establishing the Authenticity of your Host" or something?